addons/mass_mailing/migrations/13.0.2.0/openupgrade_analysis_work.txt

```diff
@@ -45,6 +45,10 @@ mass_mailing / mail.mass_mailing.list_contact_rel / list_id (many2one)
 mass_mailing / mail.mass_mailing.list   / contact_ids (many2many)       : relation is now 'mailing.contact' ('mail.mass_mailing.contact') [nothing to do]
 # NOTHING TO DO: Renamed models, but IDs don't change
 
+mass_mailing / mail.mass_mailing        / contact_list_ids (many2many)  : column1 is now 'mailing_mailing_id' ('mail_mass_mailing_id')
```

Comment:
For not having to look for the relation table, I think it would be good to include it here in the line where is mentioned the change about the columns.

Reply:
Not sure if that context is actually available.

Reply:
done